docs/players/Installation_iOS.md

```diff
@@ -4,41 +4,78 @@ You can run VCMI on iOS 12.0 and later, all devices are supported. If you wish t
 
 ## Step 1: Download and install VCMI
 
+The easiest and recommended way to install on a non-jailbroken device is to install the [AltStore Classic](https://altstore.io/) or [Sideloadly](https://sideloadly.io/). We will use AltStore as an example below. Using this method means the VCMI certificate is auto-signed automatically.
+
+i) Use [AltStore Windows](https://faq.altstore.io/altstore-classic/how-to-install-altstore-windows) or [AltStore MacOS](https://faq.altstore.io/altstore-classic/how-to-install-altstore-macos) instructions to install the store depending on the operating system you are using. 
+
+If you're having trouble enabling "sync with this iOS device over Wi-Fi" press on the rectangular shape below "Account". Example shown below.
+
+![image](https://github.com/user-attachments/assets/74fe2ca2-b55c-4b05-b083-89df604248f3)
```

Comment:
please draw an arrow to point to the button

Reply:
here's the picture

![image](https://github.com/user-attachments/assets/6590fa25-5863-45bd-b899-de37223afe33)

not sure why I wasn't able to edit the pull request

Reply:
better upload this image as a file (preferably to `images` subdirectory of the current directory) and reference it with relative path

Reply:
that's a bit too advanced for me, sorry. can't figure out how to do this

Reply:
will do, no problem

Reply:
FYI #4979